app/models/post.rb

```diff
@@ -0,0 +1,11 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
```

Comment:
- [ ] From the ERD, Kindly note that the post model also has many likes and comments. Kidly implement the relationship 👍🏽